com.krioltech.fnmap.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.krioltech.fnmap
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: fnmap
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-documents
```

Comment:
why do you need xdg-documents? The filepicker seems to support filechooser portal.

![image](https://github.com/flathub/flathub/assets/62639087/3bcc6243-bd53-4599-971a-52c41eac2e38)

Reply:
I have removed the xdg-documents privilege.